calculator/src/main/java/com/market/banica/calculator/exception/GlobalRestExceptionHandler.java

```diff
@@ -0,0 +1,27 @@
+package com.market.banica.calculator.exception;
+
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+import javax.validation.ConstraintViolationException;
+
+public class GlobalRestExceptionHandler extends ResponseEntityExceptionHandler {
+
```

Comment:
This module already has a global exception handler

Reply:
Done